api-documentation/invoice-line.md

```diff
@@ -161,3 +161,83 @@ Invoice not found.
 * `paypal`
 * `external`
 {% endhint %}
+
+## Line types
+
+The invoice lines can be of 2 basic types: increasing the invoice balance (regular lines, fees, chargebacks) and will have a positive amount attribute or decreasing the invoice balance (payments, credits) and will have a negative amount attribute. Furthermore lines can be categorized by their type (e.g.: Installment fees, Late payment fees).
+
+The type of an invoice line is determined by the `type` attributes (always present) and the optional `subtype` attributes. Below is a list of possible values.
+
+
+### Regular Lines
+
+This line type is used for lines added to the invoice at creation time, when increasing the invoice amount
+from the ClubCollect web interface or when a member adds a donation to the invoice (if enabled).
+
+| Type | Subtype | Description |
+|-|-|-|
+| `INVOICE-LINE` | | Regular Invoice line |
+| `INVOICE-LINE` | `donation` | Donation Invoice line |
+| `INVOICE-LINE` | `surcharge` | Payment process fee Invoice line |
+
+### Fees Lines
+
+At certain points of the life cycle of an invoice fees can be added to the invoice (e.g. Member initiating a payment
+in installments).
+
+
+| Type | Subtype | Description |
+|-|-|-|
+| `INSTALLMENT-FEE` | | Fee members must pay when paying for an invoice in installments |
+| `CHARGEBACK-FEE` | | When a payment is charged back a fee will be added to the invoice |
+| `LATE-PAYMENT-FEE` | | Late payment fee added when reminders are sent |
+
+### Payment Lines
+
+When payments are made (either directly by member or external payments added from the Club Collect web interface)
+payment invoice lines will are added to the invoice.
+
+
+| Type | Subtype | Description |
+|-|-|-|
+| `PAYMENT` | | Payments towards the regular invoice lines |
+| `PAYMENT` | `donation` | Payments towards donation invoice lines |
+| `JEUGDFONDS` | `jsf` | Jeugdfonds payment |
+| `PAYMENT-INSTALLMENT-FEE` | | Payments for installment fees |
+| `PAYMENT-CHARGEBACK-FEE` | | Payments for chargeback fees |
+| `PAYMENT-LATE-PAYMENT-FEE` | | Payments for late payment fees |
+| `PAYMENT-PENALTY-FEE` | | Payment for penalty fees (Legacy) |
+
+
+## Chargeback Lines
+
+When a previously successful payment is charged back (usually happens for Sepa Direct Direct or Card payments)
+Club Collect will generate chargeback lines for the registered payment lines.
+
+| Type | Subtype | Description |
+|-|-|-|
+| `CHARGEBACK` | | Chargeback line for regular invoice lines |
+| `CHARGEBACK` | `donation` | Chargeback line for donation invoice lines |
+| `CHARGEBACK-INSTALLMENT-FEE` | | Chargeback for installment fees |
+| `CHARGEBACK-LATE-PAYMENT-FEE` | | Chargeback for late payment fee |
+| `CHARGEBACK-CHARGEBACK-FEE` | | Chargeback for paid chargeback fees |
+| `CHARGEBACK-PENALTY-FEE` | | Chargeback for penalty fees |
+
+## Credit Lines
+
+When credits are added to an invocie or fees are credited from the Club Collect Web Interface we will add
```

Comment:
I'm not convinced about this wording, it's a bit overcomplicated and giving some details that are unnecessary. Let's keep it simple 

```suggestion
Credit lines are added to the invoice to discount an amount from the amount outstanding. 
```